packages/members-api/lib/users.js

```diff
@@ -146,6 +158,8 @@ module.exports = function ({
         get,
         destroy,
         getStripeSubscriptions,
+        setComplimentarySubscription,
+        cancelComplimentarySubscription,
```

Comment:
Exposed these 2 as a part of `api.members.XXX` so it's consistent with `getStripeSubscriptions` and don't have to introduce any new things in the `api.` level